config/routes.rb

```diff
@@ -4,4 +4,6 @@
   resources :articles do
     resources :comments
   end
+
+  resolve("article") { [:articles] }
```

Comment:
Why you need this?

Reply:
References to this article https://guides.rubyonrails.org/routing.html#using-resolve
I tested if it will work in my project and forgot to delete

Anyway, I don't need it.